lib/python/variable.cpp

```diff
@@ -90,44 +90,58 @@ of variances.)");
            "Rename dimensions.")
       .def_property_readonly("dtype", &Variable::dtype)
       .def(
-          "__radd__", [](Variable &a, double &b) { return a + b * units::one; },
+          "__radd__",
+          [](const Variable &a, const double &b) { return a + b * units::one; },
```

Comment:
This looks really odd. Should the `double` params be passed by value instead?

Reply:
Yes. But I didn't change that here...